solr/modules/jwt-auth/src/java/org/apache/solr/security/jwt/JWTAuthPlugin.java

```diff
@@ -847,9 +851,11 @@ protected String generateAuthDataHeader() {
     Map<String, Object> data = new HashMap<>();
     data.put(
         JWTIssuerConfig.PARAM_AUTHORIZATION_ENDPOINT, primaryIssuer.getAuthorizationEndpoint());
```

Comment:
```suggestion
        "authorization_endpoint", primaryIssuer.getAuthorizationEndpoint());
```
See comment in line below

Reply:
For this line I realize that it's been like this since 9.0, so let's not change that in this PR...

Reply:
Okay, I am leaving as it is both _PARAM_AUTHORIZATION_ENDPOINT_ and _PARAM_TOKEN_ENDPOINT_  for now, is that what you meant?

Reply:
Yea, let’s re-visit this later..